frontend/src/components/VContentLink/VContentLink.vue

```diff
@@ -1,7 +1,9 @@
 <template>
   <!-- We 'disable' the link when there are 0 results by removing the href and setting aria-disabled. -->
+  <!-- TODO: Confirm with @obulat and @dhruvkb that the above comment was inaccurate but that aria-disabled should indeed be set below -->
   <VLink
     :href="hasResults ? to : undefined"
+    :aria-disabled="!hasResults"
```

Comment:
I need to remove this change and update the comment. It seems that this is out of date with some changes that moved the disabling behaviour into VLink itself.